### ISSUE TITLE
libxmp-lite cmake is broken

### ISSUE THREAD

**Issue by isage** [libxmp/libxmp#596]
libxmp-lite cmake requires some cmake scripts that are not in repo and instead generated by Makefile.lite in libxml. Which also configure, and that requires autotools.
```
CMake Error at CMakeLists.txt:8 (include):
  include could not find requested file:

    /home/isage/dev/vita/mine/libxmp/lite/cmake/libxmp-checks.cmake


CMake Error at CMakeLists.txt:32 (include):
  include could not find requested file:

    /home/isage/dev/vita/mine/libxmp/lite/cmake/libxmp-sources.cmake


-- Configuring incomplete, errors occurred!
```

**Comment by sezero** [comment]
It is not broken, it isn't generated until `make -f Makefile.lite` is run, i.e. a lite tarball is generated.

_Maybe_ (just maybe) after #595 is finalized, we can unify things, which possibly won't be easy.

**Comment by isage** [comment]
Well, yes. But running `make -f Makefile.lite` from repo clone requires configure, which requires autotools, which kinda breaks whole point of using cmake.
Non-lite version doesn't seem to have that problem

**Comment by sezero** [comment]
You know, this is a actually dup of https://github.com/libxmp/libxmp/issues/502

And as I said, I _might_ try unifying full+lite build systems if  it's ever doable for
both cmake and autotools after #595, but no promises (TM).

**Comment by isage** [comment]
Oh, sorry, i should've checked closed issues too.
It would be okay as-is if there was release tarball with cmake support, so i guess I'll just have to wait till 4.5.1

**Comment by sezero** [comment]
Closing in favor of #676
